.vsts-ci.yml

```diff
@@ -23,339 +26,267 @@ variables:
   - group: DotNet-Installer-SDLValidation-Params
   - name: _PublishUsingPipelines
     value: true
-
 - name: _InternalRuntimeDownloadArgs
-  value: ''
-
-- ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-  - group: DotNetBuilds storage account read tokens
```

Comment:
There was logic to only set this stuff of if it is running internally. Now, this is only running internally.